FlorenceBML/BMLacq681/BMLacq681.xml

```diff
@@ -35,9 +36,139 @@ schematypens="http://relaxng.org/ns/structure/1.0"
+                                        <item xml:id="q13" n="13">
+                                            <dim unit="leaf">8</dim>
+                                            <locus from="96r" to="103v"/>
+                                        </item> 
+                                        <item xml:id="q14" n="14">
+                                            <dim unit="leaf">8</dim>
+                                            <locus from="104r" to="111v"/>
+                                        </item> 
+                                        <item xml:id="q15" n="15">
+                                            <dim unit="leaf">2</dim>
+                                            <locus from="112r" to="113v"/>
+                                        </item> 
+                                    <item xml:id="q16" n="16">
+                                        <dim unit="leaf">6</dim>
+                                        <locus from="114r" to="119v"/>
+                                        s.l. 2, stub after 3
+                                        s.l. 3, stub after 3
+                                        s.l. 5, no stub,
+                                        s.l. 6, no stub
+                                        <note>Contrary to Marrassini, who considered leave 5 (<locus target="#118"/>) and leave 6 (<locus target="#119"/>) separate from 
+                                            <ref target="#q16"/>. The text in this quire seemingly shows no breaks.</note>       
+                                    </item>
+                                    <item xml:id="q17" n="17">
+                                        <dim unit="leaf">3</dim>
+                                        <locus from="120r" to="122v"/>
+                                        s.l. 1, stub after 3
+                                    </item>
+                                    <item xml:id="q18" n="18">
+                                        <dim unit="leaf">9</dim>
+                                        <locus from="123r" to="131v"/>
+                                        s.l. 7, stub before 1
+                                        s.l. 8, stub before 1
+                                        s.l. 9, stub before 1
+                                    </item>
+                                    </list>
+                                </collation>
+                                <condition key="good">Water damage throughout. 
+                                    <locus target="#2"/> belonged originally to the position before <locus target="#7"/>.</condition>
+                            </supportDesc>
+                        </objectDesc>
+                        <handDesc>
+                            <handNote xml:id="h1" script="Ethiopic">
+                                <ab type="script"/>
+                                <date notBefore="1700" notAfter="1862">18-19th century</date>
```

Comment:
Without an explanation, I would say the span of the dating time is too large, more than 1,5 centuries. Why?

Reply:
I cannot remember to have added anything on the hands. The dating apparently is based on the paleaography. But there is a dating to the year 1728 mentioned in the manuscript on folio 122r and also in the description by Marrassini.

Reply:
Should I delete this Dating and write a new date? Or should I specify the dating and give reason for this or that dating.